tutorials/Working_with_fNIRS_data.py

```diff
@@ -0,0 +1,124 @@
+"""
+.. _tut_fnirs_data
+
+Use MNE with fNIRS data
+=======================
+# Author: Matteo Caffini  
+# Contact: <matteo.caffini@unitn.it>
+
+"""
+import numpy as np
+
+import mne
+
+raw_ndarray = np.load('/home/matteo/raw_ndarray.npy')
```

Comment:
Can you make a PR to add this to `mne.datasets.misc` [here](https://github.com/mne-tools/mne-misc-data)? It will be necessary for `CircleCI` to run, and for anybody to run your tutorial.